src/localServerApi.js

```diff
@@ -30,11 +30,34 @@ const createLocalServerApi = testkit => {
       res.sendStatus(200)
     })
     app.post(`/api/${project}/envelope/`, (req, res) => {
-      const { type, payload } = parseEnvelopeRequest(req.body)
-      if (type === 'transaction') {
-        testkit.transactions().push(transformTransaction(payload))
+      if (req.headers['transfer-encoding'] === 'chunked') {
+        let rawData = ''
+
+        req.on('data', chunk => {
```

Comment:
@mlmmn can you elaborate on this? what has changed in Sentry from v6 to v7 that now we need to gather chunks?
also, are chunks needed in a testing environment?

Reply:
I don't have an exact answer what changed in the very core of the library in that regard (and I assume this depends on 3rd party deps) - I rather simply analyzed new request data and when it comes to V7, it definitely does not contain plain body anymore.
That's why I came up with this condition and chunk support :) The `else` block is for backwards compatibility (as said earlier)